src/App.jsx

```diff
@@ -33,21 +34,23 @@ export default function App() {
   });
   return (
     <BackendUrlProvider backendUrlData={BACKEND_URL}>
-      <Router>
-        <Routes>
-          <Route path="/" element={<Landing user={user} />} />
-          <Route path="/login" element={<Login setUser={setUser} />} />
-          <Route path="/signup" element={<Signup />} />
-          <Route path="/clicky" element={<ClickyConfigurator />} />
-          <Route path="/models" element={<Models />} />
-          <Route path="/model" element={<SingleModel />} />
-          <Route path="/cart-checkout" element={<CartCheckoutPage />} />
-          <Route path="/success-checkout" element={<SuccessCheckoutPage />} />
-          <Route element={<PrivateRoutes />}>
-            {/* all protected routes here */}
-          </Route>
-        </Routes>
-      </Router>
+      <CookiesProvider>
```

Comment:
Actually i tihnk i might have wrapped cookies provider on the outside in index.js